### ISSUE TITLE
Overall stats don't appear after launch completes [v5]

### ISSUE THREAD

**Issue by iamtech123** [reportportal/reportportal#908]
**Describe the bug**
No summary stats (total, passed, failed etc.) are appearing for my launch

**To Reproduce**
Issue occurs when Test teardown or Suite Teardown is used. Tests are updated when Test teardown is removed.

**Screenshots**
![ReportPortal_TestResults_NotUpdated](https://user-images.githubusercontent.com/55270886/78383151-8c546580-75f5-11ea-89ca-03703f87e96e.jpeg)

**Comment by evjlobanova** [comment]
@iivanou could you please look at this issue? May a problem be related to the python agents?

**Comment by iivanou** [comment]
@iamtech123 are you using robotframework-reportportal for reporting your tests? If yes, what os the version of the agent?

**Comment by iamtech123** [comment]
Yes i am using robotframework-reportportal. Unfortunately python got uninstalled due to stability issue but i have used the first version of robotframework-reportportal released for Report portal5. Please refer here https://github.com/reportportal/client-Python/issues/66#issuecomment-597497455

**Comment by iivanou** [comment]
Then, how do you report your results?

**Comment by iamtech123** [comment]
The python got uninstalled last Friday. Till that time i was using it and faced the above mentioned error.

**Comment by iivanou** [comment]
The mentioned issue has already been fixed. As far as you are not using the agent for reporting, I do not understand your problem. How do you report your results if you are not using the agent?

**Comment by iamtech123** [comment]
@iivanou  Thanks a lot for the updated. As i mentioned here https://github.com/reportportal/reportportal/issues/908#issuecomment-609086333, I was using python agent till last Friday. I am setting up the environment now and will update you the results.

**Comment by iamtech123** [comment]
@iivanou i am still facing the issue. Please find the pip list below
![image](https://user-images.githubusercontent.com/55270886/78646269-6c16f600-78d6-11ea-866d-225b19a8ae4a.png)

In Report portal
![image](https://user-images.githubusercontent.com/55270886/78646387-98cb0d80-78d6-11ea-962e-76b06bb6b1f2.png)

command used
robot --listener robotframework_reportportal.listener --variable RP_UUID:"xxx" --variable RP_ENDPOINT:"http://xx:8080" --variable RP_LAUNCH:"Prj1_smoke" --variable RP_PROJECT:"prj1"  C:\Git\Sample1.robot

**Comment by iamtech123** [comment]
Sample test case (Not working)

*** Settings ***
Library           Collections

*** Test Cases ***
Case1
		LOG		Case1
    [Teardown]    LOG	END

Above test case reports correctly while i remove the [Teatdown] line
